install_edl.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+EDL_DIR=$DIR/edl
+
```

Comment:
the setup should also be part of the normal extract_tools script so you only have to run one thing to get setup